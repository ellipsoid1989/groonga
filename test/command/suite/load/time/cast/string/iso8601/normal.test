table_create Logs TABLE_NO_KEY
column_create Logs created_at COLUMN_SCALAR Time

load --table Logs
[
{"created_at": "2000-01-01T10:00:00Z"},
{"created_at": "2000-01-01t10:00:00z"},
{"created_at": "2000-01-01T10:00:00+00:00"},
{"created_at": "2000-01-01T11:01:00+01:01"},
{"created_at": "2000-01-01T21:11:00+11:11"},
{"created_at": "2000-01-01T08:59:00-01:01"},
{"created_at": "1999-12-31T22:49:00-11:11"},
{"created_at": "2000-01-01T11:00:00+01"},
{"created_at": "2000-01-01T09:00:00-01"},
{"created_at": "2000-01-01T11:01:00.123+01:01"},
{"created_at": "2000-01-02T09:00:00+23:00"},
{"created_at": "2000-01-01T10:59:00+00:59"},
{"created_at": "2000/01/01T10:00:00Z"}
]
select Logs --limit -1
