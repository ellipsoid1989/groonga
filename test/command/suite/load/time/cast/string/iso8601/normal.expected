table_create Logs TABLE_NO_KEY
[[0,0.0,0.0],true]
column_create Logs created_at COLUMN_SCALAR Time
[[0,0.0,0.0],true]
load --table Logs
[
{"created_at": "2000-01-01T10:00:00Z"},
{"created_at": "2000-01-01t10:00:00z"},
{"created_at": "2000-01-01T10:00:00+00:00"},
{"created_at": "2000-01-01T11:01:00+01:01"},
{"created_at": "2000-01-01T21:11:00+11:11"},
{"created_at": "2000-01-01T08:59:00-01:01"},
{"created_at": "1999-12-31T22:49:00-11:11"},
{"created_at": "2000-01-01T11:00:00+01"},
{"created_at": "2000-01-01T09:00:00-01"},
{"created_at": "2000-01-01T11:01:00.123+01:01"},
{"created_at": "2000-01-02T09:00:00+23:00"},
{"created_at": "2000-01-01T10:59:00+00:59"},
{"created_at": "2000/01/01T10:00:00Z"}
]
[[0,0.0,0.0],13]
select Logs --limit -1
[
  [
    0,
    0.0,
    0.0
  ],
  [
    [
      [
        13
      ],
      [
        [
          "_id",
          "UInt32"
        ],
        [
          "created_at",
          "Time"
        ]
      ],
      [
        1,
        946720800.0
      ],
      [
        2,
        946720800.0
      ],
      [
        3,
        946720800.0
      ],
      [
        4,
        946720800.0
      ],
      [
        5,
        946720800.0
      ],
      [
        6,
        946720800.0
      ],
      [
        7,
        946720800.0
      ],
      [
        8,
        946720800.0
      ],
      [
        9,
        946720800.0
      ],
      [
        10,
        946720800.123
      ],
      [
        11,
        946720800.0
      ],
      [
        12,
        946720800.0
      ],
      [
        13,
        946720800.0
      ]
    ]
  ]
]
